_posts/2025-08-15-spec-driven-development-in-action.adoc

```diff
@@ -0,0 +1,115 @@
+---
```

Comment:
we should probably target next week and update file name e.g. August 18 / 19

Reply:
Moved to August 19.